specs/PushNotifications/PushNotifications-spec.md

```diff
@@ -88,34 +88,36 @@ Power Management compliance.
 ```cpp
 int main()
 {
+    PushNotificationRegistrationOptions pushOptions = PushNotificationManager::GetPushActivatorOptions(true);
```

Comment:
IDL changes will be needed in the doc. Also please make sure the idl compiles when you change it in the doc.